frog/imports/client/StudentView/Runner.jsx

```diff
@@ -163,15 +163,12 @@ const Runner = ({ path, activity, sessionId, object, paused }) => {
   );
 
   return (
-    <MosaicWindow
-      toolbarControls={[<div key={1} />]}
-      draggable={false}
+    <ActivityWindow
       key={activity._id}
-      path={path}
```

Comment:
What was this props `path` that you remove?

Reply:
'path' was a prop for the React Mosaic library that's used to expand split windows.

Reply:
ok :)